api/server.go

```diff
@@ -75,10 +103,16 @@ func (a *API) runLoadAgentHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 	if err = lt.Run(); err != nil {
-		http.Error(w, err.Error(), http.StatusInternalServerError)
+		writeResponse(w, http.StatusInternalServerError, &APIResponse{
```

Comment:
This is another detail but if we try to run the agent twice without stopping I think it's more like a bad request from the api user and not a real problem with the server.